src/lpython/parser/tokenizer.re

```diff
@@ -315,6 +318,12 @@ int Tokenizer::lex(Allocator &al, YYSTYPE &yylval, Location &loc, diag::Diagnost
 
             // Tokens
             newline {
+                if (last_token == yytokentype::TK_COMMA
+                    || last_token == yytokentype::TK_LPAREN
+                    || (last_token == yytokentype::TK_NAME && cur[0] == ')')) {
+                    continue;
+                }
```

Comment:
```python
from x import (
    a,
    b
)
```
But the following doesn't work:
from x import ( # Comment
    a, # Comment
    b # Comment
)

Reply:
I think we have to track parentheses, and if we are inside a parentheses, we do not emit indents. Here is how CPython tracks parentheses: https://github.com/python/cpython/blob/b156578bd63a94fe3d80a4c6a8dbc067feba5d06/Parser/tokenizer.c#L2012

Reply:
After implementing
```
MAX_PAREN_LEVEL=200
char parenstack[MAX_PAREN_LEVEL]
size_t parenlevel
```
Then replace the above with: `if (parenlevel > 0) continue` or something like that.